docs/concepts/decorators-and-auto-logging.md

```diff
@@ -0,0 +1,126 @@
+(decorators-and-auto-logging)=
+# Decorators and auto-logging
+
+While it is possible to log results and artifacts using {ref}`the MLRun execution context<mlrun-execution-context>`, it is often more convenient to use the {py:func}`mlrun.handler` decorator.
+
+## Basic example
+
+Assume you have the following code in `train.py`
+
+``` python
+import pandas as pd
+from sklearn.svm import SVC
+
+def train_and_predict(train_data,
+                      predict_input,
+                      label_column='label'):
+
+    x = train_data.drop(label_column, axis=1)
+    y = train_data[label_column]
+
+    clf = SVC()
+    clf.fit(x, y)
+
+    return list(clf.predict(predict_input))
+```
+
+With the `mlrun.handler` the python function itself would not change, and logging of the inputs and outputs would be automatic. The resultant code would be as follows:
```

Comment:
With the `mlrun.handler` the python function itself does not change, and logging of the inputs and outputs is automatic. The resultant code is as follows: